### DISCUSSION TITLE
New features in Pulsebook page

### DISCUSSION THREAD

**Discussion by piekny27** [piekny27/PulseBook#13] [Ideas]
Hey guys. If you have any idea for new features, share it with us!

Team Pulsebook

**Comment by adrianbeys** [comment]
I have an idea for a new section. Addition of temperature measurement. Adding such a module will diversify the project.

**Comment by OlexTix** [comment]
I suggest adding an option to change the font color of heart monitor rate display (e.g from blue to white, red, yellow etc.)
